TweetMate/src/main/java/com/ybsystem/tweetmate/databases/PrefSystem.java

```diff
@@ -1,13 +1,21 @@
 package com.ybsystem.tweetmate.databases;
 
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.util.Log;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'android.util.Log'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AUselessImportCheck)